### PR TITLE
Allow to set include paths in LSP.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by Marenz — APPROVED** [review]
If we want to keep the TODO comments, then LGTM

**Comment by chriseth** [comment]
Please don't approve a PR that changes an external interface and has zero tests.

**Comment by chriseth** [comment]
Replaced by https://github.com/ethereum/solidity/pull/12994

**Pull request closed without merging**
